opm/autodiff/StandardWellsDense_impl.hpp

```diff
@@ -162,17 +159,17 @@ namespace Opm {
 
 
 
-    template<typename FluidSystem, typename BlackoilIndices, typename ElementContext, typename MaterialLaw>
+    template<typename TypeTag>
     template <typename Simulator>
     void
-    StandardWellsDense<FluidSystem, BlackoilIndices, ElementContext, MaterialLaw>::
+    StandardWellsDense<TypeTag>::
     assembleWellEq(Simulator& ebosSimulator,
                    const double dt,
                    WellState& well_state,
                    bool only_wells)
     {
-        const int np = wells().number_of_phases;
         const int nw = wells().number_of_wells;
+        const int numComponent = numEq();
```

Comment:
variable name is confusing!? this continues to happen below, but I won't add remarks on it from here on.